send-tokens.js

```diff
@@ -81,11 +81,11 @@ async function main() {
   // send the transaction!
   try {
     // encodes signed transaction to serialized Borsh (required for all transactions)
-    const serializedTx = signedTransaction.encode();
+    const signedSerialized = signedTransaction.encode();
```

Comment:
```suggestion
    const signedSerializedTx = signedTransaction.encode();
```
Let's keep transaction as a part of this variable's name.